package.json

```diff
@@ -55,5 +39,21 @@
     "winston": "^3.2.1",
     "xmlhttprequest": "^1.8.0",
     "zxcvbn": "^4.4.2"
-  }
+  },
```

Comment:
Always merge from dev branch first and then raise a PR to avoid unnecessary changes to dev branch,

Reply:
I merged from dev branch first, but that extra line space came by mistake by me. I am always aware to avoid unnecessary changes, but my bad this one came into the picture. Will not happen again.